turtlebot2: #namespace
    task_and_robot_environment_name: 'MultiRobotCatch-v0'
    ros_ws_abspath: "/home/lab/igal_ws"
    running_step: 0.04 # amount of time the control will be executed
    pos_step: 0.016     # increment in position for each command
    
    #qlearn parameters
    alpha: 0.1
    gamma: 0.99
    epsilon: 1.0
    epsilon_min: 0.1
    nepisodes: 10000
    nsteps: 10000
    running_step: 0.06 # Time for each step
